contracts/cw20-merkle-airdrop/README.md

```diff
@@ -60,15 +61,40 @@ pub struct InstantiateMsg {
 ```rust
 pub enum ExecuteMsg {
   UpdateConfig {
-    owner: Option<String>,
+    new_owner: Option<String>,
+    new_cw20_address: Option<String>,
+    new_native_token: Option<String>,
   },
   RegisterMerkleRoot {
     merkle_root: String,
+    expiration: Option<Expiration>,
+    start: Option<Scheduled>,
+    total_amount: Option<Uint128>,
+    hrp: Option<String>,
   },
   Claim {
     stage: u8,
     amount: Uint128,
     proof: Vec<String>,
+    /// Enables cross chain airdrops.
+    /// Target wallet proves identity by sending a signed [SignedClaimMsg](SignedClaimMsg)
+    /// containing the recipient address.
+    sig_info: Option<SignatureInfo>,
+  },
+  Burn {
+      stage: u8,
+  },
+  /// Withdraw the remaining tokens after expire time (only owner)
+  Withdraw {
```

Comment:
With recent PR, we added `Pause`. `WithdrawAll` and `BurnAll` messages. would be good to add them as well